osmaxx/api_client/API_client.py

```diff
@@ -4,6 +4,8 @@
 
 logger = logging.getLogger(__name__)
 
+HTTPError = requests.HTTPError
```

Comment:
And flake is right, it isn't used. Why not import it then at the place it actually _is_ used?